mne/beamformer/_dics.py

```diff
@@ -413,29 +925,28 @@ def csd_shapes(x):
 
 
 @verbose
-def tf_dics(epochs, forward, noise_csds, tmin, tmax, tstep, win_lengths,
-            freq_bins, subtract_evoked=False, mode='fourier', n_ffts=None,
-            mt_bandwidths=None, mt_adaptive=False, mt_low_bias=True, reg=0.05,
-            label=None, pick_ori=None, real_filter=False, verbose=None):
+def tf_dics(epochs, forward, tmin, tmax, tstep, win_lengths,
```

Comment:
should we keep this as it is but deprecate it and recommend to use make_dics, apply_dics in future version?

Reply:
It's true that this function basically runs `make_dics` and `apply_dics_csd` in a loop. However, the loop is a bit hairy to set up manually (you need to do a sliding window), so I guess this function still has a place in our API. The number of parameters is kind of insane though.

Reply:
ok true. I did not realize there was this loop over sliding window. Fair enough